config/rbac/role.yaml

```diff
@@ -102,6 +102,7 @@ rules:
   resources:
   - clusterserviceversions
   verbs:
+  - delete
```

Comment:
Is it a cluster scoped resource? I mean, don't we create the CSV for NVIDIA GPU Operator in the same namespace as the GPU Add-on? Hence, couldn't this be part of the Role, instead of the ClusterRole?

Reply:
That's correct, it is a namespaced resource and the permissions are part of the Role and not the ClusterRole: https://github.com/rh-ecosystem-edge/nvidia-gpu-addon-operator/blob/create-owned-crs-without-alm-examples/config/rbac/role.yaml#L100